spec/features/userCanSeeListOfReceivedMessages_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
```

Comment:
You don't have to require rails_helper if you require that in your `.rpsec` file instead